changelog.d/pr-9422

```diff
@@ -0,0 +1,12 @@
+synopsis: Add separate cache for getPkgConfigDb
+packages: cabal-install
+prs: #9422
+issues: #8930
+
+description: {
+  Querying pkg-config for the version of every module can be a very expensive
+  operation on some systems. This change adds a separate, per-project, for
```

Comment:
A separate *cache*?

(I think this word is also missing from the PR description.)

Reply:
Cut&pasted from the commit message which is also wrong :sob: :relaxed: